src/__tests__/__snapshots__/snapshots.test.js

```diff
@@ -0,0 +1,93 @@
+import TestRenderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../../redux/configureStore';
+import Crypto from '../../components/Crypto';
+import EFTs from '../../components/EFTs';
+import Home from '../../components/Home';
+import Forex from '../../components/Forex';
+import Loading from '../../components/Loading';
+import Majors from '../../components/Majors';
+import Stocks from '../../components/Stocks';
+
+describe('test screenshots for all components', () => {
+//   const getState = () => ({ price: 0.145947, ticket: 'EUR/USD' });
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.